dataeng/jobs/analytics/UserLocationByCourse.groovy

```diff
@@ -13,6 +13,7 @@ class UserLocationByCourse {
     public static def job = { dslFactory, allVars ->
         allVars.get('ENVIRONMENTS').each { environment, env_config ->
             dslFactory.job("user-location-by-course-$environment") {
+                description('It enables an Opsgenie Heartbeat, sets the expected duration and adds an entry of user location.')
```

Comment:
Job which computes the number of users in each country that are enrolled in each course.